warehouse/warehouse.go

```diff
@@ -955,6 +963,14 @@ func (wh *HandleT) monitorUploadStatus() {
 	})
 }
 
+func (wh *HandleT) resetInProgressJobs() {
+	sqlStatement := fmt.Sprintf(`UPDATE %s SET in_progress=%t WHERE destination_type='%s' and in_progress=%t and status!='%s' and status!='%s'`, warehouseutils.WarehouseUploadsTable, false, wh.destType, true, ExportedData, Aborted)
```

Comment:
Is it required to skip exported_data and aborted jobs?

Reply:
No, Its not required.